### ISSUE TITLE
Firefox fails to import account information, then deletes all accounts: "No accounts found for the given search query"

### ISSUE THREAD

**Issue by jacobcantele** [MetaMask/metamask-extension#8939]
**Describe the bug**
After importing my seedphrase into FireFox, I am prompted on the homescreen to import my MetaMask account preferences. I chose to import these, but then all of my accounts were deleted from the accounts list and I am unable to switch accounts from the account I was already on. Clicking the "..." button in the top right corner crashes the extension.

**To Reproduce (REQUIRED)**
Steps to reproduce the behavior, libraries used with version number, and/or any setup information to easily reproduce:

1. Import a seedphrase previous on Chrome and Mobile to the Firefox Extension.
2. When prompted in the extension to recover account details, accept the recovery.
3. See that all accounts have been deleted from the account list, and a new error shows in their place: "No accounts found for the given search query"
4. Click the "..." button in the top right of the homescreen and see that the whole extension crashes.

**Screenshots**

**Browser details (please complete the following information):**
 - OS: Manjaro Linux
 - Browser Firefox 77
 - MetaMask Version 8.0.3

**Additional context (Error Messages, etc.)**
```
Message: E is undefined
Code: TypeError
Stack:
y@moz-extension://0b8f33dd-4eb3-4dae-913f-d4052c3c5caf/ui.js:1:2840570
ha@moz-extension://0b8f33dd-4eb3-4dae-913f-d4052c3c5caf/ui-libs.js:1:997898
ks@moz-extension://0b8f33dd-4eb3-4dae-913f-d4052c3c5caf/ui-libs.js:1:1043163
Mu@moz-extension://0b8f33dd-4eb3-4dae-913f-d4052c3c5caf/ui-libs.js:1:1028710
ju@moz-extension://0b8f33dd-4eb3-4dae-913f-d4052c3c5caf/ui-libs.js:1:1028633
xu@moz-extension://0b8f33dd-4eb3-4dae-913f-d4052c3c5caf/ui-libs.js:1:1025973
Ji/<@moz-extension://0b8f33dd-4eb3-4dae-913f-d4052c3c5caf/ui-libs.js:1:984303
require<[1008]</n.unstable_runWithPriority@moz-extension://0b8f33dd-4eb3-4dae-913f-d4052c3c5caf/ui-libs.js:1:1283550
Xi@moz-extension://0b8f33dd-4eb3-4dae-913f-d4052c3c5caf/ui-libs.js:1:984012
Ji@moz-extension://0b8f33dd-4eb3-4dae-913f-d4052c3c5caf/ui-libs.js:1:984250
Qi@moz-extension://0b8f33dd-4eb3-4dae-913f-d4052c3c5caf/ui-libs.js:1:984183
se@moz-extension://0b8f33dd-4eb3-4dae-913f-d4052c3c5caf/ui-libs.js:1:1053235
Cn@moz-extension://0b8f33dd-4eb3-4dae-913f-d4052c3c5caf/ui-libs.js:1:967836
u@moz-extension://0b8f33dd-4eb3-4dae-913f-d4052c3c5caf/ui.js:1:390663

```

**Comment by jacobcantele** [comment]
This issue may be related to 3box backup. I went into the settings menu on Firefox and saw that my 3box backup was listed as disabled, although it was enabled on Chrome.

I enabled it, and then restarted the browser. When I reopened MetaMask, all of my accounts had returned.

**Comment by Gudahtt** [comment]
I believe this was fixed by #9422, so I'll close this for now.
